packages/web/src/components/composer-input/ComposerInput.tsx

```diff
@@ -0,0 +1,363 @@
+import { ChangeEvent, useCallback, useEffect, useRef, useState } from 'react'
+
+import { useAudiusLinkResolver } from '@audius/common/hooks'
+import { UserMetadata } from '@audius/common/models'
+import { splitOnNewline } from '@audius/common/utils'
+import {
+  LoadingSpinner,
+  SendIcon,
+  Text,
+  TextProps,
+  useTheme
+} from '@audius/harmony'
+
+import { TextAreaV2 } from 'components/data-entry/TextAreaV2'
+import { audiusSdk } from 'services/audius-sdk'
+import { env } from 'services/env'
+
+import { AutocompleteText } from './components/AutocompleteText'
+import { ComposerInputProps } from './types'
+
+// TODO: Need to update to remove mentions when they are no longer present in the text area value
```

Comment:
does this not work yet? so deleting a mention doesn't actually delete it? probably shouldn't merge until then?

Reply:
No it all works properly, I just wanted to remove the mention for the internal list of mentions, but not a big deal. I'll remove the comments